_config.yml

```diff
@@ -69,30 +69,37 @@ sponsors:
   - name: University of Southampton
     link: //www.southampton.ac.uk/
     image: /images/sponsors/university-of-southampton.svg
+    logo_width: default
 
   - name: Adventurous Machines
     link: https://adventurousmachines.com/?ref=studentrobotics
     image: /images/sponsors/adventurous-machines.svg
+    logo_width: default
 
   - name: Mythic Beasts
     link: https://www.mythic-beasts.com/
     image: /images/sponsors/mythic-beasts.svg
+    logo_width: default
 
   - name: Systemetric - Hills Road Robotics
     link: https://systemetric.hr-robocon.org/
     image: /images/sponsors/systemetric-hills-road.svg
+    logo_width: default
 
   - name: ITDev
     link: https://www.itdev.co.uk/
     image: /images/sponsors/itdev.png
+    logo_width: 200px
```

Comment:
how does this scale with mobile etc?

Reply:
![image](https://user-images.githubusercontent.com/1496834/160712142-68b8e686-ae5b-4822-8e3e-e215723138fd.png)